volumes.cpp

```diff
@@ -1,6 +1,7 @@
 #include <iostream>
 #include <map>
 #include <cfloat>
+#include <math.h>
```

Comment:
I was using atan2 and cos at one point, I think that I forgot to take out the include when I found a different way.